### PR TITLE
Avatar component

### LINKED ISSUE #3: Build Avatar Component
Reference: https://flowbite.com/docs/components/avatar/

### PR DESCRIPTION
**Pull request by mouracamila**
Hi! This PR adds the avatar component 

/closes #3

### CODE REVIEW

**Comment by bacali95** [comment]
Hello @mouracamila, thanks for your contribution :smile: 
I assume this PR is still in progress, so please set it as `Ready for review` once you are done.

**Comment by mouracamila** [comment]
Hi, @bacali95 I'd like to ask for a review. 
It still needs two types of avatar to build, dropdown and stacked, this is alit bit more complex, I believe this is a need made in other Issues

File: src/components/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}) => {
+  return (
+    <div className="flex items-center space-x-4">
+      <div className="relative">
+        {img ? (
+          <img
+            className={classNames(sizeClasses[size], {
+              rounded: !rounded,
+              'rounded-full': rounded,
+              'rounded-full p-1 ring-2 ring-gray-300 dark:ring-gray-500': bordered,
```
Comment (bacali95) [inline src/components/Avatar.tsx:52]: I think we can have a bordered avatar but not rounded, right?

**Review by bacali95 — CHANGES_REQUESTED** [review]
Hey @mouracamila thanks again for your contribution, this looks good for me, I just left one minor comment if you can check it :smile: 
About the dropdown feature, I will do it in this PR so we have everything covered at once, I hope you don't mind that.

**Review by bacali95 — APPROVED** [review]

**Pull request merged**